src/kademlia/node.cpp

```diff
@@ -1152,4 +1157,14 @@ void node::incoming_request(msg const& m, entry& e)
 		return;
 	}
 }
+
+void node::update_counters()
+{
+	const dht_storage_counters& dht_cnt = m_storage->counters();
+	m_counters.set_value(counters::dht_torrents, dht_cnt.torrents);
```

Comment:
I think I will rename the method update_ht_storage_counters() to reflect the fact that only the storage related counters are updated. m_counters is used in other places inside node.cpp to track other counters.